cns/restserver/util.go

```diff
@@ -766,17 +766,13 @@ func (service *HTTPRestService) SendNCSnapShotPeriodically(ctx context.Context,
 	}
 }
 
-func (service *HTTPRestService) validateIPConfigsRequest(
-	ipConfigsRequest cns.IPConfigsRequest,
-) (cns.PodInfo, types.ResponseCode, string) {
-	if service.state.OrchestratorType != cns.KubernetesCRD && service.state.OrchestratorType != cns.Kubernetes {
-		return nil, types.UnsupportedOrchestratorType, "ReleaseIPConfig API supported only for kubernetes orchestrator"
-	}
-
-	if ipConfigsRequest.OrchestratorContext == nil {
-		return nil,
-			types.EmptyOrchestratorContext,
-			fmt.Sprintf("OrchastratorContext is not set in the req: %+v", ipConfigsRequest)
+func (service *HTTPRestService) validateIPConfigsRequest(ctx context.Context, ipConfigsRequest cns.IPConfigsRequest) (cns.PodInfo, types.ResponseCode, string) {
```

Comment:
should we rename to validateSecondaryIPConfigsRequest or validateDelegatedIPConfigsRequest?

Reply:
this is the parent validateIPConfigsRequest function, if you look below, it will loop through all the validators, which are the infra one and the delegated one (if added)